provider/replacements.json

```diff
@@ -1383,6 +1383,46 @@
       "new": "~> **NOTE:** **Do not** use both `aws_default_route_table` to manage a default route table **and** `aws_main_route_table_association` with the same VPC due to possible route conflicts. See aws_default_route_table documentation for more details."
+    },
+    {
+      "old": "For more information, see the Amazon VPC User Guide on [Route Tables][aws-route-tables]. For information about managing normal route tables in Terraform, see [`aws_route_table`][tf-route-tables].\n",
+      "new": "For more information, see the Amazon VPC User Guide on [Route Tables][aws-route-tables]. For information about managing normal route tables in Pulumi, see [`aws_route_table`][tf-route-tables].\n"
+    },
+    {
+      "old": "For more information, see the Amazon VPC User Guide on [Route Tables][aws-route-tables]. For information about managing normal route tables in Terraform, see [`aws_route_table`][tf-route-tables].\n",
+      "new": "For more information, see the Amazon VPC User Guide on [Route Tables][aws-route-tables]. For information about managing normal route tables in Pulumi, see [`aws_route_table`][tf-route-tables].\n"
+    },
+    {
+      "old": "For more information, see the Amazon VPC User Guide on [Route Tables][aws-route-tables]. For information about managing normal route tables in Terraform, see [`aws_route_table`][tf-route-tables].\n",
+      "new": "For more information, see the Amazon VPC User Guide on [Route Tables][aws-route-tables]. For information about managing normal route tables in Pulumi, see [`aws_route_table`][tf-route-tables].\n"
+    },
+    {
+      "old": "For more information, see the Amazon VPC User Guide on [Route Tables][aws-route-tables]. For information about managing normal route tables in Terraform, see [`aws_route_table`][tf-route-tables].\n",
+      "new": "For more information, see the Amazon VPC User Guide on [Route Tables][aws-route-tables]. For information about managing normal route tables in Pulumi, see [`aws_route_table`][tf-route-tables].\n"
+    },
+    {
+      "old": "For more information, see the Amazon VPC User Guide on [Route Tables][aws-route-tables]. For information about managing normal route tables in Terraform, see [`aws_route_table`][tf-route-tables].\n",
+      "new": "For more information, see the Amazon VPC User Guide on [Route Tables][aws-route-tables]. For information about managing normal route tables in Pulumi, see [`aws_route_table`][tf-route-tables].\n"
+    },
+    {
+      "old": "For more information, see the Amazon VPC User Guide on [Route Tables][aws-route-tables]. For information about managing normal route tables in Terraform, see [`aws_route_table`][tf-route-tables].\n",
+      "new": "For more information, see the Amazon VPC User Guide on [Route Tables][aws-route-tables]. For information about managing normal route tables in TODO, see [`aws_route_table`][tf-route-tables].\n"
```

Comment:
It would be nice to fix the `TODO`s. They won't show up in the generated documentation, but the block that contains `terraform` will be omitted. It seems like this is the kind of repeated pattern that could be fixed by a rule in `resources.go`.

Reply:
There were four TODOs added at first, which I edited to Pulumi. It looks like after some other runs a few more were added. They all seem to be equal though?

> For more information, see the Amazon VPC User Guide on [Route Tables][aws-route-tables]. For information about managing normal route tables in Terraform, see [`aws_route_table`][tf-route-tables].\n

Can you help me understand why?

Reply:
We can merge as is. Please open an issue in this repo that describes the problem so we can track the fix later.

Reply:
https://github.com/pulumi/pulumi-aws/issues/2958